monitoring/src/main/resources/web/js/init.js

```diff
@@ -1,17 +1,18 @@
 
     function frontendApp() {
-        return akka.viz.frontend.FrontendApp()
+        return akkaviz.frontend.FrontendApp()
     }
 
     var graph = Viva.Graph.graph();
 
     var nodeSize = 40;
-    var idealLength = 5 * nodeSize;
+    var idealLength = 5.5 * nodeSize;
     var layout = Viva.Graph.Layout.forceDirected(graph, {
         springLength: idealLength,
-        springCoeff: 0.0008,
-        dragCoeff: 0.02,
-        gravity: -1.2
+        springCoeff: 0.001,
+        dragCoeff: 0.09,
+        gravity: -2.5,
+        stableThreshold: 0.1
     });
 
     var graphics = Viva.Graph.View.svgGraphics();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["Viva" is not defined.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2099574923/issues/source?bid=3121282&fileBranchId=3142760#l18)